wallet/src/wallet.rs

```diff
@@ -391,6 +392,29 @@ pub fn parse_command(
 
 type ProcessResult = Result<String, Box<dyn error::Error>>;
 
+fn check_account_for_fee(
+    rpc_client: &RpcClient,
+    config: &WalletConfig,
+    lamports_per_signature: u64,
+) -> Result<(), Box<dyn error::Error>> {
+    check_account_for_multiple_fees(rpc_client, config, lamports_per_signature, 1)
+}
+
+fn check_account_for_multiple_fees(
+    rpc_client: &RpcClient,
+    config: &WalletConfig,
+    lamports_per_signature: u64,
+    tx_count: u64,
+) -> Result<(), Box<dyn error::Error>> {
+    let balance = rpc_client.retry_get_balance(&config.keypair.pubkey(), 5)?;
+    if let Some(lamports) = balance {
+        if lamports >= lamports_per_signature * tx_count {
```

Comment:
Is there a way we could get at the `Message` for the transaction?  Would be super nice if we can just invoke https://github.com/solana-labs/solana/blob/8a64e1ddc3050add32311499cdb5a1c06c4c6501/sdk/src/fee_calculator.rs#L126 instead of essentially replicating that fee logic here (even though it's currently trivial)

Reply:
Good idea! I'll work that up

Reply:
Done; I think it's better? @mvines ?

Reply:
Much!